zerver/lib/api_test_helpers.py

```diff
@@ -453,6 +453,14 @@ def get_raw_message(client, message_id):
     validate_against_openapi_schema(result, '/messages/{message_id}', 'get',
                                     '200')
 
+    # test it was actually updated
+    url = 'messages/' + str(message_id)
+    result = client.call_endpoint(
+        url=url,
+        method='GET'
+    )
+    assert result['result'] == 'success'
+
```

Comment:
Er, I'm not sure this is actually needed. Also, this doesn't really validate the update, it just checks whether the message is retrievable. To validate the update, you have to check the message's content.